scripts-available/CDB_ColumnNames.sql

```diff
@@ -3,11 +3,12 @@ CREATE OR REPLACE FUNCTION CDB_ColumnNames(REGCLASS)
 RETURNS SETOF information_schema.sql_identifier
 AS $$
 
-    SELECT column_name 
-      FROM information_schema.columns 
-      WHERE
-        table_name IN (SELECT CDB_UserTables())
```

Comment:
That's a good point.

I guess the intention was to filter by user tables, the problem is that CDB_UserTables returns `setof name` so when there was more than one table with same name in different schemas it was selecting columns from all of them.

I'm adding a couple of asserts in the test to validate that.